### PR TITLE
Add fix for llvm/llvm-project#61402

### PR DESCRIPTION
**Pull request by gmarkall**
This addresses an issue that is triggered by Numba 0.57 - see https://github.com/numba/numba/issues/8738

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by h-vetinari** [comment]
Doesn't numba + LLVM 14 also need the SVML patches? I have an open PR for that that didn't get any comments: #192

Let's see if this compiles a bit faster after a rerender:
@conda-forge-admin, please rerender

**Comment by gmarkall** [comment]
This is unrelated to SVML. I don't know much about SVML but it's not required for Aarch64. So #192 should be considered completely independently of this.

**Comment by jakirkham** [comment]
Think Graham's point here is simply this particular change has already gone upstream (and was merged). So may be more easily integrated than the other patch (orthogonal to this one), which hasn't quite made it upstream yet.

**Comment by h-vetinari** [comment]
Yeah, I got that. I just wanted to confirm what's necessary for numba or not. I haven't had time to double-check with the upstream PR (can do sometime in the next ~12h), but if you want to go ahead and merge, I don't mind!

**Comment by jakirkham** [comment]
Happy to wait 12hrs 🙂

**Review by h-vetinari — APPROVED** [review]
Was on a phone yesterday which made this hard to check again. Thanks for the fix!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ source:
     # - patches/amd-roc-2.7.0.diff
     - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
     - patches/no-windows-symlinks.patch
+    - patches/0001-RuntimeDyld-RuntimeDyldELF-Clear-GOTOffsetMap-when-r.patch
 
 build:
   number: 2
```
Comment (h-vetinari) [inline recipe/meta.yaml:20]:
```suggestion
  number: 3
```

**Comment by jakirkham** [comment]
No worries. Been there done that 😅

Thanks Axel! 🙏

**Comment by gmarkall** [comment]
Many thanks for bumping the build number and the merge!

**Pull request merged**